x-pack/plugin/otel-data/src/main/resources/component-templates/semconv-resource-to-ecs@mappings.yaml

```diff
@@ -71,6 +71,30 @@ template:
               k8s.pod.uid:
                 type: keyword
                 ignore_above: 1024
+              k8s.container.name:
+                type: keyword
+                ignore_above: 1024
+              k8s.cronjob.name:
+                type: keyword
+                ignore_above: 1024
+              k8s.job.name:
+                type: keyword
+                ignore_above: 1024
+              k8s.statefulset.name:
+                type: keyword
+                ignore_above: 1024
+              k8s.daemonset.name:
+                type: keyword
+                ignore_above: 1024
+              k8s.replicaset.name:
+                type: keyword
+                ignore_above: 1024
+              k8s.node.uid:
```

Comment:
Nit: maybe sort this alphabetically, so that `k8s.node.uid` is next to `k8s.node.name`.